### PR TITLE
pachi 12.60

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by carlocab** [comment]
`license "GPL-2.0-or-later`?

https://github.com/pasky/pachi/search?q=license

File: Formula/pachi.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pachi < Formula
   desc "Software for the Board Game of Go/Weiqi/Baduk"
   homepage "https://pachi.or.cz/"
-  url "https://github.com/pasky/pachi/archive/pachi-12.50.tar.gz"
-  sha256 "62c8d44bd4610fe9534a1f21bb092da209c9fb8dcb8d39558d79adabe31e740a"
+  url "https://github.com/pasky/pachi/archive/pachi-12.60.tar.gz"
+  sha256 "3c05cf4fe5206ba4cbe0e0026ec3225232261b44e9e05e45f76193b4b31ff8e9"
   license "GPL-2.0"
```
Comment (carlocab) [inline Formula/pachi.rb:6]:
```suggestion
  license "GPL-2.0-or-later"
```

**Comment by chenrui333** [comment]
that does not look like the license file declaration for the library.

**Comment by chenrui333** [comment]
cc @Rylan12

**Comment by carlocab** [comment]
🤷 on their homepage they link to that repo for the source code and searching their repo for the word `license` just returns GPL 2 or later.

**Comment by Rylan12** [comment]
Hmm, not sure. It kinda looks like the `GPL-2.0-or-later` that comes up when you search is only for [that file (`sgflib.py`)](https://github.com/pasky/pachi/blob/38fba7ac0f71d19123803d09c7a1daab8772ce86/tools/sgflib/sgflib.py). I'm not sure we can assume that the rest of the repo is `GPL-2.0-or-later`. Maybe we want to ask upstream?

**Comment by carlocab** [comment]
Would be useful to ask upstream I guess. However, my understanding from reading previous discussions about this (in particular FX's comments -- not tagging to avoid useless notifications) is that the license file tells you the general classification (GPL, BSD, etc). But to know which specific kind of license it is, you look at the headers, then choose the most restrictive license classification based on that.

This process gives you GPL-2.0-or-later, I believe.

**Comment by carlocab** [comment]
I think I remember reading it here: https://github.com/Homebrew/homebrew-core/issues/58225

**Pull request closed without merging**